internal/config/stub.go

```diff
@@ -10,24 +10,7 @@ import (
 )
 
 func NewBlankConfig() *ConfigMock {
-	defaultStr := `
-# What protocol to use when performing git operations. Supported values: ssh, https
-git_protocol: https
-# What editor gh should run when creating issues, pull requests, etc. If blank, will refer to environment.
-editor:
-# When to interactively prompt. This is a global config that cannot be overridden by hostname. Supported values: enabled, disabled
-prompt: enabled
-# A pager program to send command output to, e.g. "less". Set the value to "cat" to disable the pager.
-pager:
-# Aliases allow you to create nicknames for gh commands
-aliases:
-  co: pr checkout
-# The path to a unix socket through which send HTTP connections. If blank, HTTP traffic will be handled by net/http.DefaultTransport.
-http_unix_socket:
-# What web browser gh should use when opening URLs. If blank, will refer to environment.
-browser:
-`
-	return NewFromString(defaultStr)
+	return NewFromString(defaultConfigStr)
```

Comment:
Oh that's kinda neat that this already had been duplicated here, indicating that it was already a bit of an application concern.